app/views/chats/_message.html.haml

```diff
@@ -0,0 +1,7 @@
+- date = message.created_at.strftime('%H:%M')
+- text = message.text
```

Comment:
Try not to set new variables inside partials. Just access the `message` variable.